src/index.d.ts

```diff
@@ -748,29 +748,29 @@ declare namespace admin.firestore {
 
 declare namespace admin.instanceId {
   interface InstanceId {
-    app: admin.app.App;
+    readonly app: admin.app.App;
 
     deleteInstanceId(instanceId: string): Promise<void>;
   }
 }
 
 declare namespace admin.projectManagement {
   interface ShaCertificate {
```

Comment:
`ShaCertificate` is used as input in `addShaCertificates()` and `deleteShaCertificate()`, but I still keep the readonly here as I don't think user will change the object after initializing it

Reply:
@nbegley do you concur?

Reply:
I concur - `ShaCertificate`s are intended to be immutable.